templates/library/status.html

```diff
@@ -132,22 +132,22 @@ <h3 class="modal-title">
                                         <i class="mdi mdi-delete"></i>
                                     </a>
                                 </div>
-                                <div class="col-md-8 labels">
-                                    <label class="col-md-5 col-sm-5">Quality</label>
-                                    <label class="col-md-5 col-sm-5">Status</label>
-
-                                </div>
 
                                 <div id="movie_settings" class="col-md-8 btn-group">
-                                        <select id="movie_quality" class="btn btn-default col-md-5 col-sm-5">
+                                  <div>
+                                  <label>Quality</label>
+                                        <select id="movie_quality" class="btn btn-default">
                                             %for i in profiles:
                                             <option value="${i}">${i}</option>
                                             %endfor
                                         </select>
-                                        <select id="movie_status" class="btn btn-default col-md-5 col-sm-5">
+                                  </div><div>
```

Comment:
Set all indentation to 4 spaces and make sure everything is nested correctly. I'm a stickler for neat formatting.

This doesn't work great on smaller screens. But it is an easy fix. On each plain <div> element add the class `col-md-5` and `col-sm-12`. Then they'll take a full row width on small screens but 5/12th of a row on medium/large screens, so the `col-md-2` save button fits in nicely.

Then you'll probably notice that the selects don't fit perfectly. Change the css for them to have `width: 100%` and they'll look better on any size.